AGENTS.md

```
@@ -43,16 +43,26 @@ Use the appropriate tier based on where you are in the development cycle:
    Runs all query datasets (stocks, unordered_stocks, smart_homes, taxis, ordered_bluesky, unordered_bluesky) via `PyOfflineServer` and compares output against expected results. Requires pycer to be built first (`scripts/build_pycer.sh`).
 
 4. **E2E tests with sanitizers** (run alongside tier 3):
-   Build pycer with sanitizers and run e2e tests:
+   Build pycer with sanitizers and run e2e tests. Example with address sanitizer:
    ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)